.github/dependabot.yml

```diff
@@ -6,6 +6,8 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    reviewers:
+      - "dockstore/dockstore"        
```

Comment:
Looks like there a couple of people in that dockstore/dockstore team we wouldn't want/expect to do reviews -- does this assign everybody or round-robin reviewers?

Reply:
Hmmm, it depends on whether this respects the code review settings on the team

![Screenshot from 2022-11-15 16-57-26](https://user-images.githubusercontent.com/1730679/202033699-08a0966b-31a9-418d-9c5f-2871257ca17a.png)